changelog/fix-use-type-is-in-filter

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+A Support type_is_in filter for Transactions page URL
```

Comment:
Optional nitpick - Since the changelog is public facing, adding a bit more description and context may be helpful. 

```suggestion
Support 'type_is_in' filter for Transactions list report, to allow easy filtering by multiple types. 
```

Reply:
Thanks for catching that @nagpai . And to be clear, we should always strive to make our changelogs easy for anyone to understand what's changed and why it's good (not just "public" ones).